pkg/policies/dataplane/iface/limiter.go

```diff
@@ -28,4 +29,5 @@ type Limiter interface {
 	GetSelector() *selectorv1.Selector
 	RunLimiter(labels map[string]string) *flowcontrolv1.LimiterDecision
 	GetLimiterID() LimiterID
+	GetHistogram(labels map[string]string) prometheus.Observer
```

Comment:
Its not really a Histogram but a Summary. [`prometheus.Observer`](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Observer) is the common interface for both histograms and summary. Can rename accordingly.

Reply:
Ok